crds/AzureBackendPool.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: azurebackendpools.appgw.ingress.azure.io
+spec:
+  group: appgw.ingress.azure.io
+  versions:
+    - name: v1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            spec:
+              type: object
+              properties:
+                backendPoolInfo:
+                  description: "Object storing the lastest information about backend pool such as back pool id and ip addresses"
+                  type: array
+                  items:
+                    type: object
+                    properties:
+                      backendPoolID:
+                        description: "Subresource ID of the backend pool of the Application gateway"
+                        type: string
+                      ipAddresses:
+                        description: "A list of Pod IP addresses"
+                        type: array
+                        items:
+                          type: string
+  scope: Cluster
+  names:
+    plural: azurebackendpools
```

Comment:
should this be application gateway backend pool ? (so as not to confuse with SLB's backend pool)

Reply:
> should this be applicationgatewaybackendpool ? (so as not to confuse with SLB's backend pool)
Better name :)